src/test/java/com/cloudant/tests/CloudantClientTestBase.java

```diff
@@ -0,0 +1,82 @@
+package com.cloudant.tests;
+
+import com.cloudant.client.api.CloudantClient;
+
+import org.junit.After;
+import org.junit.Before;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Created by Rhys Short on 21/05/15.
+ */
+public abstract class CloudantClientTestBase {
```

Comment:
I always find our testing hierarchies just make it difficult to understand what a particular test has set up and has available to it.